### ISSUE TITLE
Activity Stream descriptions not translated

### ISSUE THREAD

**Issue by sshedmake** [ansible/awx#3828]
As per https://github.com/ansible/tower/issues/2061#issuecomment-416343623 , https://github.com/ansible/tower/issues/2061#issuecomment-417378475, https://github.com/ansible/tower/issues/2061#issuecomment-417419818 opening remaining issue as separate ticket.

i.e.

Post second round of translations, tested this bug report with Japanese google-chrome.

Now, most of the bullets (as listed in comment zero of this bug report) appear localized in Japanese, except for the following screen(s) (appear in English):

and 

as per https://github.com/ansible/tower/issues/2061#issuecomment-401374808 , i request to re-consider the screenshots attached towards following bullets (as UI appears in English for the said screens):

> (4) Tower Dashboard and interface -> Activity Stream window

-> all Actions in EVENT column:

![screenshot from 2018-06-28 12-04-03](https://user-images.githubusercontent.com/12151785/42093700-df154f36-7bca-11e8-8c87-ec5c66c6a20a.png)

-> at the right side pagination, ITEMS  1 - 20 of 99 VIEW PER PAGE ( '**of**' needs to be gettext-ed):
![screenshot from 2018-06-29 07-29-38](https://user-images.githubusercontent.com/12151785/42105085-b642ab2c-7bec-11e8-9c2d-eaa933cbfaaa.png)

_Originally posted by @sshedmake in https://github.com/ansible/tower/issues/2061#issuecomment-416343623_

**Comment by mabashian** [comment]
The event column is actually tremendously complicated for translation.  The UI takes these disparate pieces from the api: 

<img width="1297" alt="screen shot 2019-01-22 at 4 24 53 pm" src="https://user-images.githubusercontent.com/9889020/51566386-64e70200-1e62-11e9-8294-a63e068d293e.png">

and cobbles them together into a string while attempting to convert the first word to past-tense.  I'm not really sure how to even approach that one.

I can provide a fix for `ITEMS 1-20 of 22`

**Comment by mabashian** [comment]
After talking with Bill I'm going to push this one out.  This will require quite a bit of thought before we can commit to anything here.

**Comment by unlikelyzero** [comment]
@mabashian @wenottingham should this one get the `component:ui` tag?

**Comment by rooftopcellist** [comment]
@unlikelyzero there is UI and API work required here.  

I put together a quick POC [here](https://github.com/ansible/awx/compare/devel...rooftopcellist:activity_stream_translations?expand=1) for the API side.  But the PR still doesn't cover many cases.  The process of convertiong the `operation` to past tense that @mabashian mentioned is the tricky part that I don't have a solution for yet.  

In English for reference:

![image](https://user-images.githubusercontent.com/11698892/83454466-3c91ef00-a42a-11ea-800e-cbf6a1154817.png)



In Japanese:

![image](https://user-images.githubusercontent.com/11698892/83454214-e02ecf80-a429-11ea-89fd-ed75811ac7fe.png)

**Comment by unlikelyzero** [comment]
@rooftopcellist @ryanpetrello @mabashian can I moved this to 4.0 and `component:ui_next` ?

**Comment by ryanpetrello** [comment]
Yep.

**Comment by jlmitch5** [comment]
Not sure how to solve this one.  It is somewhat a unique case because we generate the strings based on pieces of data from the api, basically we take pieces of this data like: `create`, `workflow_job_template_node`, `foo` and create a description like:

"created workflow_job_template_node `<a href="/#/templates/workflow_template/1>foo<a>`"

As such, there's not really any way for us to "mark" this for translation.   we could tackle this in a piecemeal fashion.  instead of just adding "d" or "ed" to the action words we could do a big conditional with all the possible ones and then have it be like:

```
let translatedPastTenseAction;
if ('create') {
  translatedPastTenseAction = i18n._(t`created`);
} else if ('update') {
  translatedPastTenseAction = i18n._(t`updated`);
} ...
```

etc.

---

For the rersource type we could have a similar conditional that is like:

```
let translatedResourceType;
if ('workflow_job_template') {
  translatedPastTenseAction = i18n._(t`Workflow Template`);
} else if ('project') {
  translatedPastTenseAction = i18n._(t`Project`);
} ...
```

**Comment by sshedmake** [comment]
Added @Spredzy here
